docs/_posts/2015-02-18-react-conf-roundup-2015.md

```diff
@@ -0,0 +1,264 @@
+---
+title: React.js Conf Round-up 2015
+layout: post
+author: Steven Luscher
+---
+
+It was a privilege to welcome the React community to Facebook on January 28–29 for the first-ever React.js Conf, and a pleasure to be be able to unveil three new technologies that we've been using internally for quite some time: GraphQL, Relay, and React Native.
```

Comment:
Do you mean to clarify that it was _physically_ at Facebook? I'll change it to “Facebook HQ” – that should do it.